src/types/onyx/PurchaseList.ts

```diff
@@ -0,0 +1,188 @@
+/** Subscription type for a purchase */
+type Subscription = {
+    /** Whether new users are added automatically */
+    addNewUsersAutomatically?: boolean;
+
+    /** Whether the subscription auto-renews */
+    autoRenew?: boolean;
+
+    /** Date when auto-renew was last changed */
+    autoRenewLastChangedDate?: string;
+
+    /** End date of the subscription */
+    endDate?: string;
+
+    /** Start date of the subscription */
+    startDate?: string;
+
+    /** Type of subscription */
+    type?: string;
+
+    /** Number of users in the subscription */
+    userCount?: number;
+};
+
+/** Type for a billable policy */
+type BillablePolicy = {
+    /** List of actors in the policy */
+    actorList?: string;
+    /** Approved spend amounts by currency */
+    approvedSpend?: Record<string, number>;
+    /** Whether the policy is corporate */
+    corporate?: boolean;
+    /** Expensify card spend by currency */
+    expensifyCardSpend?: Record<string, number>;
+    /** Type of the policy */
+    type?: string;
+};
+
+/** Message type for a purchase */
+type Message = {
+    /** Account manager account ID */
+    accountManagerAccountID?: number;
+
+    /** List of approved accountant account IDs */
+    approvedAccountantAccountIDs?: number[];
+
+    /** Approved spend amounts by currency */
+    approvedSpend?: Record<string, number>;
+
+    /** Billable amount */
+    billableAmount?: number;
+
+    /** Billable amount before free trial discount */
+    billableAmountBeforeFreeTrialDiscount?: number;
+
+    /** Record of billable policies with their details */
+    billablePolicies?: Record<string, BillablePolicy>;
+
+    /** Billing type */
+    billingType?: string;
+
+    /** Card spend surcharge percentage */
+    cardSpendSurchargePercent?: number;
+
+    /** Cash back amount */
+    cashBackAmount?: number;
+
+    /** Cash back percentage */
+    cashBackPercentage?: number;
+
+    /** Chat only actor list */
+    chatOnlyActorList?: string;
+
+    /** Corporate actor count */
+    corporateActorCount?: number;
+
+    /** Corporate revenue */
+    corporateRevenue?: number;
+
+    /** Expensify card monthly spend */
+    expensifyCardMonthlySpend?: number;
+
+    /** Expensify card spend by currency */
+    expensifyCardSpend?: Record<string, number>;
+
+    /** Free actor count */
+    freeActorCount?: number;
+
```

Comment:
```suggestion
```
We got rid of this plan so while it may be present on older purchases I don't think we need to document it